android/filament-android/src/main/java/com/google/android/filament/TransformManager.java

```diff
@@ -199,6 +200,33 @@ public int getParent(@EntityInstance int i) {
         return nGetParent(mNativeObject, i);
     }
 
+
+    /**
+     * Returns the number of children of an {@link EntityInstance}.
+     *
+     * @param i the {@link EntityInstance} of the transform component to query.
+     * @return The number of children of the queried component.
+     */
+    public int getChildCount(@EntityInstance int i) {
+        return nGetChildCount(mNativeObject, i);
+    }
+
+    /**
+     * Gets a list of children for a transform component.
+     *
+     * @param i the {@link EntityInstance} of the transform component to get the children from.
+     * @param count The maximum number of children to retrieve.
+     * @return Array of retrieved children {@link Entity}.
+     */
+    @Entity
+    @NonNull
+    public  int[] getChildren(@EntityInstance int i, @IntRange(from = 1) int count) {
```

Comment:
Nit: remove extra space after `public`.

There should also be an API that takes an int[] passed by the caller so allocations can be avoided. This method can call the other one then, passing a `new int[]`.